<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VncPreferences</class>
 <widget class="QWidget" name="VncPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>396</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="vncGroupBox">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="connectionLabel">
        <property name="text">
         <string>Connection &amp;type:</string>
        </property>
        <property name="buddy">
         <cstring>kcfg_Quality</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="kcfg_Quality">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>0</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>Use this to specify the performance of your connection. Note that you should select the speed of the weakest link - even if you have a high speed connection, it will not help you if the remote computer uses a slow modem. Choosing a level of quality that is too high on a slow link will cause slower response times. Choosing a lower quality will increase latencies in high speed connections and results in lower image quality, especially in 'Low Quality' mode.</string>
        </property>
        <item>
         <property name="text">
          <string>High Quality (LAN, direct connection)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium Quality (DSL, Cable, fast Internet)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Low Quality (Modem, ISDN, slow Internet)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QCheckBox" name="kcfg_Scaling">
          <property name="text">
           <string>Scale to Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="resolutionDummyLabel"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QVBoxLayout">
        <item>
         <widget class="KComboBox" name="resolutionComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>280</width>
            <height>0</height>
           </size>
          </property>
          <property name="whatsThis">
           <string>Here you can specify the resolution of the remote desktop. This resolution determines the size of the desktop that will be presented to you.</string>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>Minimal (640x480)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Small (800x600)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Normal (1024x768)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Large (1280x1024)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Very Large (1600x1200)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Current Screen Resolution</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Custom Resolution (...)</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout">
          <item>
           <widget class="QLabel" name="widthLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Width:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_ScalingWidth</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="kcfg_ScalingWidth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>This is the width of the remote desktop. You can only change this value manually if you select Custom as desktop resolution above.</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>800</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="heightLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>H&amp;eight:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_ScalingHeight</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="kcfg_ScalingHeight">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>This is the height of the remote desktop. You can only change this value manually if you select Custom as desktop resolution above.</string>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>600</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="use_ssh_tunnel">
     <property name="text">
      <string>Connect via SSH tunnel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ssh_groupBox">
     <property name="title">
      <string>SSH tunnel options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="use_loopback">
        <property name="text">
         <string>Tunnel via loopback address</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Port:</string>
        </property>
        <property name="buddy">
         <cstring>ssh_tunnel_port</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="ssh_tunnel_port">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User name:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="ssh_tunnel_user_name"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dont_copy_passwords">
     <property name="text">
      <string>Don't copy passwords from supported password managers to remote hosts</string>
     </property>
     <property name="toolTip">
      <string>Supported password managers (like KeePassXC) mark copied passwords with an additional MIME type "x-kde-passwordManagerHint" which is used for filtering.</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>428</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
